src/sp_sloppy.c

```diff
@@ -41,10 +41,11 @@ static void array_handler(INTERNAL_FUNCTION_PARAMETERS,
     void (*orig_handler)(INTERNAL_FUNCTION_PARAMETERS)) {
   void (*handler)(INTERNAL_FUNCTION_PARAMETERS);
   zval func_name;
-  zval params[3] = {{{0}}};
+  zval params[3];
```

Comment:
Can't we leave something like `zaval params[3] = {0}`?

Reply:
it's either memset or `zval params[3] = {{{0},{{0, 0, 0, 0}}, {0}}};`, i find memset a bit more readable

Reply:
Fair enough